tests/test_cases/test_cocotb/test_deprecated.py

```diff
@@ -86,3 +88,41 @@ async def example():
         pass
     with assert_deprecated():
         cocotb.hook()(example)
+
+
+async def test_pack_deprecated(_):
+
+    class Example(ctypes.Structure):
+        _fields_ = [
+            ("a", ctypes.c_byte),
+            ("b", ctypes.c_byte)]
+
+    e = Example(a=0xCC, b=0x55)
+
+    with assert_deprecated():
+        a = cocotb.utils.pack(e)
+
+    assert a == bytes(e)
+
+
+@cocotb.test()
+async def test_unpack_deprecated(_):
+
+    class Example(ctypes.Structure):
+        _fields_ = [
+            ("a", ctypes.c_byte),
+            ("b", ctypes.c_byte)]
+
+    e = Example(a=0xCC, b=0x55)
+    f = Example(a=0xCC, b=0x55)
+
+    b = b'\x01\x02'
+
+    with assert_deprecated():
+        cocotb.utils.unpack(e, b)
+
+    assert e.a == 1 and e.b == 2
+
+    ctypes.memmove(ctypes.pointer(f), b, ctypes.sizeof(f))
+
+    assert f.a == 1 and f.b == 2
```

Comment:
These tests would make more sense in the pytest tests - we don't need GPI to run them.

Reply:
Done.